gammapy/modeling/model.py

```diff
@@ -59,3 +59,27 @@ def from_dict(cls, data):
         for parameter in init.parameters.parameters:
             setattr(init, parameter.name, parameter)
         return init
+
+
+def make_model(tag, *args, **kwargs):
```

Comment:
As I already said, I care more about consistency, than the actual name. So I would prefer to introduce it as `create_model()` for now, to keep the consistency with the existing `create_crab_spectral_model` and `create_cosmic_ray_spectral_model` (admittedly those were decided without extensive discussion by @JouvinLea , but following one of your comments https://github.com/gammapy/gammapy/issues/1788#issuecomment-499476024). It would be weird to have  `make_model()` showing up in the docs here: https://docs.gammapy.org/dev/modeling/index.html#id1. It would just introduce further diversity in the naming scheme in the model namespace for no good reason. It would be even more weird, in the light of the extensive model re-naming we did for v0.14. If you feel strongly about the `make` in the name, then please also rename the other `create_` factory functions at the same time. Even if we don't have it consistent everywhere in Gammapy let us try to keep at least the newly introduced models namespace consistent.